pkg/parse/Parser_test.go

```diff
@@ -2,18 +2,17 @@ package parse
 
 import (
 	"bytes"
-	"fmt"
 	"io"
-	"io/ioutil"
 	"os"
-	"os/exec"
 	"path"
 	"strings"
 	"testing"
 
+	"github.com/pmezard/go-difflib/difflib"
```

Comment:
nit: There's no reason to separate this import from the others.

Reply:
go imports doesnt always get it right :)